README.md

```diff
@@ -203,17 +203,15 @@ There's a simple way to combine actions in Redux Zero by using the [spread opera
 
```

Comment:
You'll need to change the text, because it's saying that you need to use the spread operator from ES2015+
With this PR, that should not be the case anymore.

Reply:
done :)

Reply:
`utility` instead of `utilitary`, Am I right?